lib/modules/platform/bitbucket-server/index.ts

```diff
@@ -1214,6 +1217,152 @@ export async function mergePr({
   return true;
 }
 
+export async function expandGroupMembers(
+  reviewers: string[],
+): Promise<string[]> {
+  logger.debug(`expandGroupMembers(${reviewers.join(', ')})`);
+  const expandedUsers: string[] = [];
+
+  for (const reviewer of reviewers) {
+    const [baseEntry, modifier] = reviewer.split(':');
+
+    if (isReviewerGroup(baseEntry)) {
+      const groupName = baseEntry.replace(regEx('^@reviewer-group/'), '');
+      const groupUsers = await getUsersFromReviewerGroup(groupName);
+      if (groupUsers.length === 0) {
+        continue;
+      }
+
+      if (modifier === undefined) {
+        expandedUsers.push(...groupUsers); // Add all users from the group
+        continue;
+      }
+      const randomCount = parseModifier(modifier);
+      if (randomCount === null) {
+        expandedUsers.push(...groupUsers);
+        continue;
+      }
+      expandedUsers.push(...sampleSize(groupUsers, randomCount));
```

Comment:
```suggestion
      if (modifier) {
        const randomCount = parseModifier(modifier);
        if (randomCount) {
          expandedUsers.push(...sampleSize(groupUsers, randomCount));
          continue;
        }
      }

      expandedUsers.push(...groupUsers);
```

Inverting the logic makes it more explicit what is going on and also removes yet another `expandedUsers.push(...groupUsers); continue;`